serializer/serializer.c

```diff
@@ -550,6 +550,17 @@ status_t send_mam_req_deserialize(const char* const obj, ta_send_mam_req_t* req)
     goto done;
   }
 
+  json_result = cJSON_GetObjectItemCaseSensitive(json_obj, "prng");
+  if (json_result != NULL) {
+    size_t prng_size = strlen(json_result->valuestring);
+
+    if (prng_size != 81) {
```

Comment:
is it better to use `MAM_PRNG_SECRET_KEY_SIZE / 3` instead of `81`?

Reply:
I'm not a fan of create another macro. If you insist I can see if there's a similar one.

Reply:
That macro is existed in the current `entangled`

Reply:
This is not the same definition in mam prng layer. It indicate the seed to generate prng, not the result.

Reply:
OK got it